### PR TITLE
Bump to BFG 1.12.3

### PR DESCRIPTION
**Pull request by rtyley**

### CODE REVIEW

File: Library/Formula/bfg.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Bfg < Formula
   homepage "https://rtyley.github.io/bfg-repo-cleaner/"
-  url "https://repo1.maven.org/maven2/com/madgag/bfg/1.12.0/bfg-1.12.0.jar"
+  url "https://repo1.maven.org/maven2/com/madgag/bfg/1.12.3/bfg-1.12.3.jar"
   sha1 "665b78b35a501adc49816e0553d643b56f05043f"
```
Comment (jacknagel) [inline Library/Formula/bfg.rb]: The checksum needs to be updated.

Comment (rtyley) [inline Library/Formula/bfg.rb]: Apologies, updated.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `bfg 1.12.3`.

**Comment by rtyley** [comment]
> For future reference the preferred commit message format for simple version updates is bfg 1.12.3.

Roger that - will remember that for the future.

**Pull request closed without merging**